<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteFinder</class>
 <widget class="QWidget" name="RouteFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>144</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TheBestWay</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose source graph</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="LoadGraph">
       <property name="text">
        <string>Load from file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NewDirRB">
       <property name="text">
        <string>Create new directed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NewUndirRB">
       <property name="text">
        <string>Create new undirected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="proceedButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
